### ISSUE TITLE
rtx crashes when .python-version file is present

### ISSUE THREAD

**Issue by hytromo** [jdxcode/rtx#108]
I have ` .tool-versions` file at a parent directory with tool versions. In a child directory there's a legacy file, `.python-version`. When I `cd` to the child directory while I've activated my zsh shell, I get the following stacktrace (sometimes during `cd` and other times during `rtx install` to install the missing python version):

```
Error: 
   0: [python] script exited with non-zero status: no exit status
   1: No such file or directory (os error 2)

Location:
   src/plugins/script_manager.rs:150

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 3 frames hidden ⋮                               
   4: rtx::plugins::script_manager::ScriptManager::read::h3ac8e9dca7b09ecc
      at <unknown source file>:<unknown line>
   5: rtx::plugins::Plugin::parse_legacy_file::h7a473ac1ca186eef
      at <unknown source file>:<unknown line>
   6: rtx::config::config_file::legacy_version::LegacyVersionFile::parse::h1efffc5cc0eec2b6
      at <unknown source file>:<unknown line>
   7: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &mut F>::call_once::hff1d6be8f30d7350
      at <unknown source file>:<unknown line>
   8: <alloc::vec::Vec<T,A> as alloc::vec::spec_extend::SpecExtend<T,I>>::spec_extend::hcd444ea93157f5e3
      at <unknown source file>:<unknown line>
   9: rayon::iter::plumbing::bridge_producer_consumer::helper::h293c7e9f628c513c
      at <unknown source file>:<unknown line>
  10: <rayon_core::job::StackJob<L,F,R> as rayon_core::job::Job>::execute::h75c230c81aafbf72
      at <unknown source file>:<unknown line>
  11: rayon_core::registry::WorkerThread::wait_until_cold::h63ef2408225018e0
      at <unknown source file>:<unknown line>
  12: rayon_core::registry::ThreadBuilder::run::h58941d25e929444a
      at <unknown source file>:<unknown line>
  13: std::sys_common::backtrace::__rust_begin_short_backtrace::h5698d368608ccb22
      at <unknown source file>:<unknown line>
  14: core::ops::function::FnOnce::call_once{{vtable.shim}}::h9a4ef96a19d3e7cf
      at <unknown source file>:<unknown line>
  15: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::ha93239d92e7b610d
      at /rustc/fc594f15669680fa70d255faec3ca3fb507c3405/library/alloc/src/boxed.rs:2000
  16: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hebda57b0236de612
      at /rustc/fc594f15669680fa70d255faec3ca3fb507c3405/library/alloc/src/boxed.rs:2000
  17: std::sys::unix::thread::Thread::new::thread_start::h1541b4390bbc26ca
      at /rustc/fc594f15669680fa70d255faec3ca3fb507c3405/library/std/src/sys/unix/thread.rs:108
  18: start_thread<unknown>
      at <unknown source file>:<unknown line>
  19: clone<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering. 
```

Version: `rtx 1.4.0 (built 2023-02-03)`

**Comment by amoosbr** [comment]
Based on the release notes, it should be fixed with [version 1.5.1](https://github.com/jdxcode/rtx/releases/tag/v1.5.1).

**Comment by hytromo** [comment]
Yes I can confirm that this is now fixed.
